src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {

```

Comment:
The class implementation should not begin with an empty line. Please remove the empty line at the beginning of the `ExcavatorProducer` class definition.